spec/models/query/waiting_for_part_payment_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Query::WaitingForPartPayment, type: :model do
+  describe '#find' do
+
+    let(:user) { create :user }
+    let!(:application1) { create :application, user_id: user.id, office_id: user.office_id }
+    let!(:application2) { create :application, user_id: user.id, office_id: user.office_id }
+    let!(:application3) { create :application, user_id: user.id, office_id: user.office_id }
+    let!(:payment1) { create :part_payment, application: application1, expires_at: 2.days.from_now }
```

Comment:
These are not converted, but they're not actually being used as variables in the spec. We could just convert them to `before` block.